src/pages/HelloWord.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+export default function HelloWord() {
+  return (
+    <section className="h-screen grid">
+      <div className="bg-yellow-500 p-52 max-w-lg grid m-auto w-full">
+        <div className="m-auto">
+          <h2 className="text-center text-gray-700 font-bold tracking-wider">Hello Word</h2>
+        </div>
+      </div>
+    </section>
```

Comment:
You should not overthink the hello world, tracking is unnecessary here, try to keep it simple as `hello world` :)

Reply:
Sure, I remove it.